app/src/main/kotlin/pl/gov/mc/protego/ui/main/DashboardActivity.kt

```diff
@@ -23,6 +24,12 @@ class DashboardActivity : BaseActivity() {
             startActivity(Intent(this, OnboardingActivity::class.java))
             finish()
         }
+        observeLiveData(viewModel.noInternetConnection) { hasInternetConnection ->
```

Comment:
skoro metody do pokazania i chowania dialogu z brakiem internetu są w `BaseActivity` to nie powinno być tam logiki, która sprawdzi czy te dialogi pokazać? W obecnym rozwiązaniu każda nowa aktywność będzie musiała mieć w swoim modelu taką liveDate do sprawdzania internetu.

Reply:
#94